imageroot/update-module.d/15fix_6900

```diff
@@ -7,10 +8,8 @@
 # Needed to upgrade from crowdsec:1.0.6
 
 if systemctl is-active -q crowdsec-firewall-bouncer.service; then
```

Comment:
time to test is-active rather than is-active, the bouncer could be enabled but in error what do you think

Reply:
I meaned : `if systemctl is-enabled-q crowdsec-firewall-bouncer.service; then`